accounts/dam_prototype/vpcs.tf

```diff
@@ -0,0 +1,6 @@
+module "dam_prototype_vpc" {
+  source = "../modules/vpc"
+
+  vpc_name   = "dam-prototype"
+  cidr_block = "172.31.0.0/16"
+}
```

Comment:
I wonder: do we need this VPC?

I'm hoping to make a single Terraform configuration you can run `terraform plan/apply` in, and it spins up a copy of the storage service in a new account. The theory being that you could swap out the Terraform provider (it'll be `dams-admin`, but you could use your own) and it would all work for you.

It'll end up referring to the VPC module in this repo (pinned at a particular commit, for now), and using that VPC for the services. So I'm not sure I'll actually use this.

Reply:
That's fine - i'm just following the new account pattern _just in case_.